onnxruntime/core/providers/xnnpack/math/matmul.cc

```diff
@@ -13,15 +13,12 @@
 namespace onnxruntime {
 namespace xnnpack {
 
-bool MatMul::IsMatMulOnnxNodeSupported(const NodeUnit& node_unit, const GraphViewer& graph) {
+bool MatMul::IsOnnxNodeSupported(const NodeUnit& node_unit, const GraphViewer& graph) {
   bool supported = false;
   const onnxruntime::Node& node = node_unit.GetNode();
 
   // use do {} while(false) so it's easier to set a breakpoint on the return
   do {
-    const auto alpha = node.GetAttributes().find("alpha");
```

Comment:
@shalvamist  
https://github.com/onnx/onnx/blob/main/docs/Operators.md#matmul
Seems Matmul didn't have attribute "alpha", then it will return an invalid iterator and lead to crash.
Seems we need more UT for Matmul